### PR TITLE
Update Sina.xml

### PR DESCRIPTION
**Pull request by gloomy-ghost**
Mixed Content: The page at 'https://login.sina.com.cn/crossdomain2.php?action=login&entry=weibo...' was loaded over HTTPS, but requested an insecure script 'http://i.sso.sina.com.cn/js/ssologin.js'.

### CODE REVIEW

**Comment by J0WI** [comment]
This requires a valid [test url](https://github.com/EFForg/https-everywhere/blob/master/ruleset-testing.md).
Please also add a short comment in the rule file, why this exclusion is needed.

File: src/chrome/content/rules/Sina.xml [hunk 1]
```diff
@@ -139,6 +139,10 @@
 	<target host="help.sina.com.cn" />
 	<target host="licaishi.sina.com.cn" />
 	<target host="login.sina.com.cn" />
+		<!-- The page at 'crossdomain2.php' is loaded over HTTPS, but requests an insecure script 'http://i.sso.sina.com.cn/js/ssologin.js'. -->
```
Comment (J0WI) [inline src/chrome/content/rules/Sina.xml]: This only happens when the parameter `?action=login` is present.

**Comment by gloomy-ghost** [comment]
Done

**Comment by renyouguo** [comment]
+1

**Comment by gloomy-ghost** [comment]
@J0WI I think this PR has fixed #5106 #6227

**Pull request merged**